index.html

```diff
@@ -168,25 +168,25 @@ <h3>FOOTER</h3>
         </div>
         <div class="middle-last">
           <h3>BLOG POSTS</h3>
-          <div class="_last-middle">
+          <div class="last-middle">
             <img src="./assets/images/workshop.jpg" alt="" />
             <div>
               <p>Lorem</p>
               <p>Sed mattis nunc</p>
             </div>
           </div>
-          <div class="_last-middle bottom">
+          <div class="last-middle bottom">
             <img src="./assets/images/gondol.jpg" alt="" />
             <div>
               <p>Ipsum</p>
               <p>Sed mattis nunc</p>
             </div>
           </div>
```

Comment:
There is a hover property on this image section look at the original you will see what am talking about
![Screenshot from 2023-03-29 11-38-50](https://user-images.githubusercontent.com/116273730/228508878-386b5c60-1cd3-4bf4-9bb7-3664791d82d4.png)